tests/ParserTest.php

```diff
@@ -1079,7 +1079,7 @@ public function commentExtracting(): void
         self::assertSame('* Number 10 *', $fooBarComments[0]->getComment());
 
         // Media -> declaration -> rule.
```

Comment:
Is this comment still up to date? (Haven't checked, though.)

Reply:
Actually it seems to fit better following this change.  Though `RuleSet` and `DeclarationBlock` as class names are misnomers.